src/shared/components/palette/history/event-log/event-log-table.component.ts

```diff
@@ -53,4 +53,12 @@ export class EventLogTableComponent implements OnInit {
   caseHistoryClicked(eventId: string) {
     this.onCaseHistory.emit(eventId);
   }
+
+  ariaLabelText(event: CaseViewEvent): string {
```

Comment:
My only comment, but I do not feel greatly passionate about it is... most methods we have like this are name getAriaLabel or buildAriaLabel. So a bit more descriptive about what it does.

Reply:
Changed the method name to getAriaLabel